### DISCUSSION TITLE
E@SY - Possibilités d'intégration ?

### DISCUSSION THREAD

**Discussion by ABIAdrienG** [witsa/synapps#572] [Q&A] [answered]
Bonjour,

Est-ce possible d'importer Synapps dans un automate E@SY ?

Merci d'avance.
Bien à vous,

**Comment by manufitoussiwit** [comment, marked as answer]
Salut, 
Non malheureusement ce n'est pas possible : Synapps utilise des fonctionnalités qui ne peuvent pas être ajoutées dans un E@SY.

++

**Reply by ABIAdrienG** [reply]
D'accord, merci !
